src/components/dialogs/BirthDateSettings.tsx

```diff
@@ -30,42 +33,71 @@ export function BirthDateSettingsDialog({
   const t = useTheme()
   const {_} = useLingui()
   const {isLoading, error, data: preferences} = usePreferencesQuery()
+  const isBirthdateUpdateAllowed = useIsBirthdateUpdateAllowed()
 
   return (
     <Dialog.Outer control={control} nativeOptions={{preventExpansion: true}}>
       <Dialog.Handle />
-      <Dialog.ScrollableInner
-        label={_(msg`My Birthday`)}
-        style={web({maxWidth: 400})}>
-        <View style={[a.gap_sm]}>
-          <Text style={[a.text_xl, a.font_semi_bold]}>
-            <Trans>My Birthday</Trans>
-          </Text>
-          <Text style={[a.leading_snug, t.atoms.text_contrast_medium]}>
-            <Trans>
-              This information is private and not shared with other users.
-            </Trans>
-          </Text>
+      {isBirthdateUpdateAllowed ? (
+        <Dialog.ScrollableInner
+          label={_(msg`My Birthday`)}
```

Comment:
```suggestion
          label={_(msg`My Birthdate`)}
```
nit: `Birthday` ➡️ `Birthdate`